components/movieNight.py

```diff
@@ -0,0 +1,173 @@
+
+
+# view-suggestion command
+# View a specific suggestion of a specific user
+@movie_night_group.command(name="view-suggestion", description="View a specific suggestion")
+@discord.app_commands.describe(user="User's movie-suggestion list")
+@discord.app_commands.describe(movie_name="View more details on a specific movie")
+async def view_suggestion(interaction: discord.Integration, user: discord.Member, movie_name: str):
+    if not user:
+        user = interaction.user
+    reply = suggestion_database.get_suggestion_embed(user.name, movie_name)
+    await interaction.response.send_message(embed=reply, delete_after=ut.DELETE_AFTER_SECONDS)
+
+
+# pick-host command
+# Allows admins to pick the host for upcoming movie night
+# User gets reminder to pick movie everyday at noon
+@movie_night_group.command(name="pick-host", description="Pick host for upcoming movie night [Admin Only]")
+@discord.app_commands.describe(user="Host for upcoming movie night")
+@discord.app_commands.describe(prev_host="Bumps this user to the bottom of the list")
+@discord.app_commands.checks.has_role(ut.env["ADMIN_ROLE"])
+async def pick_host(interaction: discord.Integration, user: discord.Member, prev_host: discord.Member = None):
+    if failed_embed := suggestion_database.bump_prev_host(prev_host):
+        await interaction.response.send_message(ut.get_role_str("ADMIN_ROLE"), embed=failed_embed, delete_after=ut.DELETE_AFTER_SECONDS)
+        return
+    # Set host has no fail condition, so not validated
+    embed = await upcomingMovie.set_host(user.name)
+    # If prev_host exists, then bump has to be successful to reach this line
+    if prev_host:
+        embed.description += f"\n{ut.get_member_str(prev_host.name)} was successfully bumped to the end of the list!"
+
+    await interaction.response.send_message(embed=embed, delete_after=ut.DELETE_AFTER_SECONDS)
+
+
+# DEPRECATED!; Command not used enough. Removed to reduce bloat
+# reset-host command
+# Allows admins to reset the host for upcoming movie night
+# @movie_night_group.command(name="reset-host", description="Resets host for upcoming movie night, if any [Admin Only]")
+# @discord.app_commands.checks.has_role(ut.env["ADMIN_ROLE"])
+# async def reset_host(interaction: discord.Integration):
+#    embed = await upcomingMovie.reset_host()
+#    await interaction.response.send_message(embed=embed, delete_after=ut.DELETE_AFTER_SECONDS)
+
+
+# pick-movie command
+# Allows upcoming host to pick the upcoming movie from suggestion-list
+@movie_night_group.command(name="pick-movie", description="Pick movie for upcoming movie night")
+@discord.app_commands.describe(movie_name="Name of the movie from suggested list")
+async def pick_movie(interaction: discord.Interaction, movie_name: str):
+    embed = await upcomingMovie.set_movie(interaction.user.name, movie_name, suggestion_database)
+    await interaction.response.send_message(embed=embed, delete_after=ut.DELETE_AFTER_SECONDS)
```

Comment:
Let's not delete this message for the same reason as above.

Reply:
Same reply as above.